app/components/exercises/ExerciseList.js

```diff
@@ -34,7 +66,7 @@ const ExerciseList = ({ user, exercises, deleteExercise }) => (
           else return 0;
         }).map( exercise =>
           <TableRow key={ exercise.id } selectable={false} className="exercise-row">
-            <TableRowColumn style={{"paddingBottom":"1em", "paddingTop": "1em", width: "15em"}}>
+            <TableRowColumn className="img_col">
               <img src={exercise.img_url}></img>
```

Comment:
try using 'img-responsive' class. this currently is thumbnailed on mobile. should ideally take up screen width.